qoqo-macros/src/lib.rs

```diff
@@ -848,6 +848,10 @@ pub fn devicewrapper(
                 })
             }
 
+            fn __repr__(&self) -> String{
+                format!("{:?}", self.internal)
+            }
+
 
```

Comment:
I'd recommend adding a test for this, I believe there's already a bunch of "__fmt__" tests, a line could be added to check the "__repr__" function in those tests.